### PR TITLE
Show task progress while it is active

### PR DESCRIPTION
**Pull request by axe312ger**
# This pull request is based on https://github.com/contentful/contentful-batch-libs/pull/98. It should be merged first!

Display current task progress with percentage and estimated time to finish below the task. Verbose logging to CI should work pretty well.

The output of the progress observable might need a progress bar or other improvements.

Error display is not yet implemented and might cause visual issues with the update renderer, this needs some more work :)

Also, i am pretty sure the observable code can be optimised.

## Standard progress display
[asciicast](https://asciinema.org/a/651gowrqc2u200jb3jumxb9o3)

## Verbose progress display for CI
[asciicast](https://asciinema.org/a/5emr7a9re4yvjvrp6h9x98jyp)

### CODE REVIEW

**Comment by Khaledgarbaya** [comment]
@axe312ger what's up with this PR ?

**Comment by axe312ger** [comment]
We should discuss this, since the error output with the current error handling is broken.

Also, since `getFullSourceSpace` is kinda the only function used by contentful-export, we may move it to the contentful-export repo to ease up maintenance. Why? Because we are changing the API here, it is then Observable based and no more Promise based which will break custom implementations of `contentful-batch-libs`

--> https://github.com/contentful/contentful-export/search?utf8=%E2%9C%93&q=contentful-batch-libs&type=

File: .babelrc [hunk 1]
```diff
@@ -8,7 +8,9 @@
     }]
   ],
   "plugins": [
-    "add-module-exports"
+    "add-module-exports",
```
Comment (Khaledgarbaya) [inline .babelrc]: can you also remove the `"debug": true,` it's so annoying when building

**Comment by axe312ger** [comment]
This is now fully implemented via listr itself, promises and awesome loops. All without RxJS. Closing this one and deleting the branch...

**Pull request closed without merging**